src/Sylius/Behat/spec/Context/Setup/PromotionContextSpec.php

```diff
@@ -159,4 +160,29 @@ function it_creates_percentage_shipping_discount_action_for_promotion(
 
         $this->itGivesPercentageDiscountOnShippingToEveryOrder($promotion, 0.1);
     }
+
+    function it_creates_item_percentage_discount_action_for_promotion_products_with_specific_taxon(
+        $actionFactory,
+        $objectManager,
+        ActionInterface $action,
+        PromotionInterface $promotion,
+        TaxonInterface $taxon
+    ) {
+        $taxon->getCode()->willReturn('taxonCode');
```

Comment:
Let's us more realistic example, will be easier to understand: ``woman_dresses``.